engine/sql/api/263_key_disabled.sql

```diff
@@ -0,0 +1,7 @@
+
+-- +migrate Up
+ALTER TABLE project_key ADD COLUMN disabled BOOLEAN NOT NULL DEFAULT FALSE;
+UPDATE project_key SET disabled = false;
```

Comment:
DEFAULT false is not updating all existing rows ?

If there is lots of values the right pattern is:

ALTER TABLE project_key ADD COLUMN disabled BOOLEAN NOT NULL;
UPDATE project_key SET disabled = false;
ALTER TABLE "project_key" ALTER COLUMN disabled SET DEFAULT false;

Reply:
> DEFAULT false is not updating all existing rows ?

nope

there are not so much rows (1468) in the table